test/spec/http.js

```diff
@@ -162,6 +167,29 @@ describe('HTTP Requestor', () => {
         });
       });
     });
+    it('can customize okta user agent', () => {
+      createAuthClient();
+      sdk._oktaUserAgent.addEnvironment('fake/x.y');
+      return http.httpRequest(sdk, { url })
+      .then(res => {
+        expect(res).toBe(response1);
+        let expectUserAgent = `${USER_AGENT} fake/x.y`;
+        if (ENV === 'node') {
```

Comment:
It looks like there are 3 tests combined into one here.
test 1: can customize (browser)
test 2: can customize (node)
test 3: adds node version (node)

rather than declaring ENV as a global and adding a conditional in the test, lets explicitly tests our logic in OktaUserAgent by mocking `isBrowser`, and break these tests out without conditionals